### PR TITLE
Docs changes for Cody featuring Sourcegraph v5.7 release

### PR DESCRIPTION
**Pull request by MaedahBatool**
This PR tackles all the docs changes in the Sourcegraph 5.7 release that are related to Cody. It caters the following projects:

- Cody Web GA
   - Removes the Beta label from everywhere we mention
   - Add mentions of LLM selector to only ENT users using the new model configuration

- JetBrains Unified Chat UI
   - Replace all media with old UI
   - Reorder of information based on a similar structure of VS Code docs

- Smart Apply in JetBrains 
- Mentions about Prompts in docs

### CODE REVIEW

File: docs/cody/clients/cody-with-sourcegraph.mdx [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 <p className="subtitle">Learn how to use Cody in the web interface with your Sourcegraph.com instance.</p>
 
-<Callout type="info"> The new chat UI for Cody for web is currently in Beta and is available to users on Sourcegraph versions >=5.5.</Callout>
+<Callout type="info"> The new unified chat UI for Cody for web is currently available to users on Sourcegraph versions >=5.5.</Callout>
```
Comment (taylorsperry) [inline docs/cody/clients/cody-with-sourcegraph.mdx:5]: - Users must be on JB version 2023.2 or above in order to get these changes (and future updates)! 

- We’ll continue to support Cody for JB versions below 2023.2 but only with critical bug fixes. We won’t be adding new features or improvements to those older versions.

File: docs/cody/clients/cody-with-sourcegraph.mdx [hunk 2]
```diff
@@ -33,7 +33,7 @@ Sourcegraph.com users with Cody **Free** and **Pro** can choose from a list of s
 
 ![llm-select-web](https://storage.googleapis.com/sourcegraph-assets/Docs/llm-select-web-0724.jpg)
 
-Users on an Enterprise Sourcegraph instance do not have the option to choose an LLM model. Their site admin will configure the default LLM model for chat.
+Users on an Enterprise Sourcegraph instance do not have the option to choose an LLM model. Their site admin will configure the default LLM model for chat. However, Enterprise users with the new [model configuration](/cody/clients/model-configuration) can use the LLM selection dropdown to choose a chat model.
```
Comment (taylorsperry) [inline docs/cody/clients/cody-with-sourcegraph.mdx:36]: This is true, right, @aramaraju ?

File: docs/cody/clients/install-jetbrains.mdx [hunk 3]
```diff
@@ -66,122 +66,106 @@ Once Cody is successfully connected, you'll see that the sign-in panel has been
...
+At any point in time, you can edit these context chips or remove them completely if you do not want to use these as context. Any chat without a context chip will instruct Cody to use no codebase context. However, you can always provide an alternate `@-mention` file or symbols to let Cody use it as a new source of context.
 
-Users on Cody **Free** and **Pro** can choose from a list of supported LLM models for Chat and Commands. Both Cody Free and Pro users can choose from a list of supported LLM models for Chat and Commands.
+When you have both a repository and files @-mentioned, Cody will search the repository for context while prioritizing the mentioned files.
 
-![llm-selection-cody](https://storage.googleapis.com/sourcegraph-assets/Docs/jb-llm-select-0724.png)
+### Selecting Context with @-mentions
 
-Enterprise users get Claude 3 (Opus and Sonnet) as the default LLM models without extra cost. Moreover, Enterprise users can use Claude 3.5 models through Cody Gateway, Anthropic BYOK, AWS Bedrock (limited availability), and GCP Vertex.
+Cody's chat allows you to add files and symbols as context in your messages.
```
Comment (taylorsperry) [inline docs/cody/clients/install-jetbrains.mdx:147]: JB doesn't support @-symbol yet.

File: docs/cody/clients/cody-with-sourcegraph.mdx (hunk 2 above)
Comment (aramaraju) [reply docs/cody/clients/cody-with-sourcegraph.mdx:36]: We can capture that in Early Access to be super clear on that statement, but functionally accurate

File: docs/cody/clients/cody-with-sourcegraph.mdx (hunk 1 above)
Comment (kalanchan) [reply docs/cody/clients/cody-with-sourcegraph.mdx:5]: perhaps we should remove the word unified, we call it unified internally but to our customers it's just a new UI. Even "new UI" is a bit misleading, maybe refreshed Cody Web UI?

Comment (MaedahBatool) [reply docs/cody/clients/cody-with-sourcegraph.mdx:5]: In that case, let's not call it anything. The messaging should show that our chat experience is top-notch by default. Instead, I can add a callout that you need to be on the latest JB version for this new chat UI.

**Comment by MaedahBatool** [comment]
The docs of this PR now live here: https://github.com/sourcegraph/docs/pull/603

**Pull request merged**